.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,54 @@
+name: Scorecard supply-chain security
+on:
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '17 4 * * 5'
```

Comment:
Is the specific time of "4:17 AM every Friday" intentional?

Reply:
so, i did want to give it some weekly time to run at :)  do you think another time would work better?